indico/htdocs/js/custom/clipboard.js

```diff
@@ -0,0 +1,43 @@
+$(function() {
```

Comment:
Do we really put the Indico header in .js files? It is not present in the other files in `indico/htdocs/js/custom/`.